examples/cross_kms_support/catalogValidationValues.json.template

```diff
@@ -0,0 +1,9 @@
+{
+  "ibmcloud_api_key": $VALIDATION_APIKEY,
+  "region": "au-syd",
+  "resource_tags": $TAGS,
+  "prefix": $PREFIX
+  "kms_instance_guid": $KMS_INSTANCE_GUID
```

Comment:
Pipeline is being updated to support `$GE_OPS_KP_US_SOUTH_GUID` so please update to use that

Reply:
Done.